news/5306.bugfix

```diff
@@ -0,0 +1 @@
+Ignore errors copying socket files for local source installs.
```

Comment:
It might be handy to note that this is a Python 3 only fix here.

Reply:
Updated [here](https://github.com/pypa/pip/pull/6844/files#diff-36ee43f5f90b9f550cb1db618295a914R1).